fdm-app/app/components/custom/cultivation/form.tsx

```diff
@@ -0,0 +1,207 @@
+import { Button } from "@/components/ui/button"
+import { Calendar } from "@/components/ui/calendar"
+import {
+    FormControl,
+    FormDescription,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form"
+import {
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+} from "@/components/ui/popover"
+import { cn } from "@/lib/utils"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { format } from "date-fns/format"
+import { nl } from "date-fns/locale/nl"
+import { CalendarIcon } from "lucide-react"
+import { useEffect } from "react"
+import { Form } from "react-router"
+import { RemixFormProvider, useRemixForm } from "remix-hook-form"
+import type { z } from "zod"
+import { Combobox } from "../combobox"
+import { LoadingSpinner } from "../loadingspinner"
+import { FormSchema } from "./schema"
+import type { CultivationsFormProps } from "./types"
+
+export function CultivationForm({
+    b_lu_catalogue,
+    b_sowing_date,
+    b_terminating_date,
+    options,
+    action,
+}: CultivationsFormProps) {
+    const form = useRemixForm<z.infer<typeof FormSchema>>({
+        mode: "onTouched",
+        resolver: zodResolver(FormSchema),
+        defaultValues: {
+            b_lu_catalogue: b_lu_catalogue,
+            b_sowing_date: b_sowing_date,
+            b_terminating_date: b_terminating_date,
+        },
```

Comment:
the N content of harvestable crop is not changeable (yet)?

Reply:
That is related to harvest not cultivation